getting-started.md

```diff
@@ -0,0 +1,61 @@
+
+- Getting Started: A Novella
+Last updated 10/7/19
+
+- Repo Setup:
+  
+  - Note: NOT recommend for devs to use React Native Debugger. It will be too slow. We have our own system of logging in the console that works great!
+  - Note: The repo on GitHub is called phd-mobile.
+  
+  1. Create a GitHub account with your yum email. This is the email that will get the invite to our internal repos. Teams message     Stephen Miller with your new GitHub account info and he will add you.
+  2. Install Xcode and Android Studio following the directions on the React Native docs. We do not use Expo, so use React Native Cli instructions.
+  3. You need to create a personal token in Github. Go to Settings -> Developer Settings -> Personal Tokens to make one.
+  4. Fork repo. Use your username + the personal token as the password. Directions for this are below in the Forked Repos section.
+  5. Install nvm (node version manager). Realm (a dependency in our project) does not like new node. You need 10.13.0 for a good time. You can do this by going here and following the instructions for installation: https://github.com/nvm-sh/nvm
+  6. Switch node versions to 10.13.0.
```

Comment:
I didn't have to do this. Are you sure this is required for everyone?

Reply:
Sam had mentioned node versions earlier on. The Jesse said it wasn't a thing. Are you sure you have the latest node version? I was unable to yarn install with our repo with the latest node because realm reused to install. I checked Realm's docs and indeed, the latest node version does not work with it. I had to use nvm to switch node versions and it worked perfectly.

Reply:
Optionally, I can just move it BELOW the yarn install part with something like: 'If yarn install is not working properly, try this: ' and put the nvm stuff with that.